openapi/openapi.yml

```diff
@@ -1750,6 +1777,14 @@ components:
           description: Maintenance window start time
           type: integer
           nullable: true
+        read_replica:
+          description: Read replica of the Postgres database
```

Comment:
```
is_read_replica:
  description: Whether this is a read replica of the Postgres database
```

Not sure about `is_` prefix? We usually don't use `is_` prefix, but `read_replica` and `read_replicas` fields together seems confusing. @fdr, @geemus any opinions?

Reply:
considering I have removed read_replicas, this should be OK?